Makefile

```diff
@@ -113,6 +109,9 @@ build/.ran-third-party-cmake:: $(DEPS_BUILD_DIR)
 		$(CMAKE_PRG) -G '$(BUILD_TYPE)' $(BUNDLED_CMAKE_FLAG) $(BUNDLED_LUA_CMAKE_FLAG) \
 		$(DEPS_CMAKE_FLAGS) $(THIS_DIR)/third-party
 endif
+build/.ran-third-party-cmake::
+	mkdir -p build
+	touch $@
```

Comment:
does this override the definition on line 107? should this be in a `else` ?

Reply:
No.
https://www.gnu.org/software/make/manual/html_node/Double_002dColon.html
> Each double-colon rule’s recipe is executed if the target is older than any prerequisites of that rule. If there are no prerequisites for that rule, its recipe is always executed (even if the target already exists). This can result in executing none, any, or all of the double-colon rules.
>
> The double-colon rules for a target are executed in the order they appear in the makefile.